GameData/ROEngines/PartConfigs/GEM46_BDB.cfg

```diff
@@ -0,0 +1,132 @@
+PART
+{
+	module = Part
+	name = ROE-GEM46
+	author = Cobaltwolf, Vader111
+	
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+	
+	mass = 1.9919
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+	
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/Delta/bluedog_Delta_GEM46
+	}
+	
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_attach = 0.0, 0.0, -0.363, 0.0, 0.0, 1.0, 1
+	CoMOffset = 0.0, -0.18, -0.18
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0	
+	
+	title = GEM 46
+	manufacturer = Orbital ATK
+	description = The 60-inch (1.5-meter) diameter GEM 60 is used on the Delta IV in sets of two or four. When two boosters are used, both are equipped with thrust vector control (TVC). When four are used, two boosters use TVC while the other two use fixed nozzles to reduce weight. Burn time 90 seconds.
+	
+	tags = ATK, GEM46, booster, radial, solid, srb,
+	
+	engineType = GEM-46
+	stagingIcon = SOLID_BOOSTER
+	stageOffset = 1
+	childStageOffset = 1
+	
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.5
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+	breakingForce = 200
+	breakingTorque = 200
+	crashTolerance = 7
+	
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+	}
+	
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalResponseSpeed = 16
+		useGimbalResponseSpeed = true
+		gimbalTransformName = gimbalTransform
+		gimbalRange = 4.0
+	}
+	
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchNose
+		switcherDescription = Nose Shape
+		switcherDescriptionPlural = Nose Shapes
+		affectDragCubes = True
+		affectFARVoxels = True
+		SUBTYPE
+		{
+			name = Straight
+			transform = StraightNose
+		}
+		SUBTYPE
+		{
+			name = Angled
+			transform = AngledNose
+		}
+	}
+	
+	MODULE
+	{
+		name = ModuleSurfaceFX
+		thrustProviderModuleIndex = 0
+		fxMax = 1
+		maxDistance = 80
+		falloff = 2
+		thrustTransformName = thrustTransform
+	}
+	
+	MODULE
+	{
+		name = FXModuleAnimateThrottle
+		animationName = GEM60_Emit_Anim
+		dependOnEngineState = True
+		responseSpeed = 0.001
+	}
+}
+
+@PART[ROE-GEM46]:AFTER[RealismOverhaulEnginesPost] 
+{
+	@MODULE[ModuleEngineConfigs] 
+	{
+		@CONFIG[GEM-46/TVC-Ground]
+		{
+			@origMass = 1.9919
+		}
+		@CONFIG[GEM-46/Fixed-Ground]
+		{
+			@origMass = 1.9919
+		}
+		@CONFIG[GEM-46/Fixed-Air]
+		{
+			@origMass = 2.1959
+		}
```

Comment:
Still confused about this structure here instead of properly in the RO engine config.